ffc-netcore-parent/sdk/Dockerfile

```diff
@@ -1,2 +1,8 @@
 ARG NETCORE_VERSION=3.1
-FROM mcr.microsoft.com/dotnet/core/sdk:${NETCORE_VERSION}-alpine AS base
+FROM mcr.microsoft.com/dotnet/core/sdk:${NETCORE_VERSION}-alpine
```

Comment:
Should include `ENV ASPNETCORE_ENVIRONMENT=production`, the development equivalent should have development.